.props/_Nupkg.props

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- This targets file will add an English localization tag to our xml documentation file. -->
+  <Import Project="$(TargetsRoot)\AddXmlLanguage.targets" />
```

Comment:
Do we need `$(TargetsRoot)` or just using the `.props` folder for all `*.props` file?

Reply:
I honestly hadn't considered that, but I think it makes sense. I'm testing that change now.